allennlp/commands/train.py

```diff
@@ -172,7 +186,41 @@ def train_model(params: Params, serialization_dir: str, file_friendly_logging: b
     """
     prepare_environment(params)
 
-    os.makedirs(serialization_dir, exist_ok=True)
+    if os.path.exists(serialization_dir):
+        logger.info(f"Continuing from prior training at {serialization_dir}.")
+
+        continued_config_file = os.path.join(serialization_dir, CONFIG_NAME)
+        if not os.path.exists(continued_config_file):
+            logger.warning("Continuing from prior training, but no config.json was found.")
+            logger.warning("If you've changed your code since you last trained, the training may fail.")
+        else:
+            loaded_params = Params.from_file(continued_config_file)
+
+            # Check whether any of the training configuration differs from the configuration we are resuming.
+            # If so, warn the user that training may fail.
+            fail = False
+            flat_params = params.as_flat_dict()
+            flat_loaded = loaded_params.as_flat_dict()
+            for key in flat_params.keys() - flat_loaded.keys():
+                logger.warning(f"Key '{key}' found in training configuration but not in the serialization "
```

Comment:
These should be `logger.error` instead of `logger.warning`.

Reply:
Done.